cli/cmd/flags.go

```diff
@@ -64,19 +66,38 @@ func bindRPCURL(cmd *cobra.Command, rpcURL *string) {
 func bindUnjailConfig(cmd *cobra.Command, cfg *unjailConfig) {
 	netconf.BindFlag(cmd.Flags(), &cfg.Network)
 	bindPrivateKeyFile(cmd, &cfg.PrivateKeyFile)
-	_ = cmd.MarkFlagRequired("network")
+	_ = cmd.MarkFlagRequired(flagNetwork)
+}
+
+func bindDelegateValConfig(cmd *cobra.Command, cfg *delegateValConfig) {
+	netconf.BindFlag(cmd.Flags(), &cfg.network)
+	bindPrivateKeyFile(cmd, &cfg.privateKeyFile)
+
+	cmd.Flags().Uint64Var(&cfg.amount, flagDelegationAmount, cfg.amount, "increase delegation amount in OMNI (minimum 1 OMNI)")
+
+	bindExecutionRPC(cmd, &cfg.evmRPC)
+
+	_ = cmd.MarkFlagRequired(flagConsPubKeyHex)
+	_ = cmd.MarkFlagRequired(flagDelegationAmount)
+	_ = cmd.MarkFlagRequired(flagNetwork)
 }
 
 func bindCreateValConfig(cmd *cobra.Command, cfg *createValConfig) {
-	netconf.BindFlag(cmd.Flags(), &cfg.Network)
-	bindPrivateKeyFile(cmd, &cfg.PrivateKeyFile)
+	netconf.BindFlag(cmd.Flags(), &cfg.network)
+	bindPrivateKeyFile(cmd, &cfg.privateKeyFile)
+
+	cmd.Flags().StringVar(&cfg.consensusPubKeyHex, flagConsPubKeyHex, cfg.consensusPubKeyHex, "Hex-encoded validator consensus public key")
+	cmd.Flags().Uint64Var(&cfg.selfDelegation, flagSelfDelegation, cfg.selfDelegation, "Self-delegation amount in OMNI (minimum 100 OMNI)")
 
-	cmd.Flags().StringVar(&cfg.ConsensusPubKeyHex, flagConsPubKeyHex, cfg.ConsensusPubKeyHex, "Hex-encoded validator consensus public key")
-	cmd.Flags().Uint64Var(&cfg.SelfDelegation, flagSelfDelegation, cfg.SelfDelegation, "Self-delegation amount in OMNI (minimum 100 OMNI)")
+	bindExecutionRPC(cmd, &cfg.evmRPC)
 
 	_ = cmd.MarkFlagRequired(flagConsPubKeyHex)
 	_ = cmd.MarkFlagRequired(flagSelfDelegation)
-	_ = cmd.MarkFlagRequired("network")
+	_ = cmd.MarkFlagRequired(flagNetwork)
+}
+
+func bindExecutionRPC(cmd *cobra.Command, evmRPC *string) {
+	cmd.Flags().StringVar(evmRPC, "evm-rpc", "", "Omni EVM execution RPC API endpoint")
```

Comment:
`--execution-rpc` and `--consensus-rpc`. Please add a comment that that this is optional. And something like `consensus.<network>.omni.network` will be used if empty.